integrations/appium.js

```diff
@@ -0,0 +1,37 @@
+require('dotenv').config()
+
+const { Print } = require('@ianwalter/print')
+
+module.exports = class AppiumIntegration {
+  constructor (context) {
+    // Set up a print instance on the integration instance so it can be reused.
+    this.print = new Print({ level: context.logLevel })
+    this.print.debug('Appium integration enabled')
+
+    // Define the global capability options.
+    context.webdriver.port = 4723
```

Comment:
Does Appium always use this port? Not sure if it should be hard coded.

Reply:
It's the default port, but I think it would be a good idea for us to specify it in an environment variable instead - will push that change now

Reply:
Then maybe it should be conditional assignment like:
```js
    context.webdriver.port = context.webdriver.port || 4723
```